docs-2.0/6.monitor-and-metrics/rocksdb-statistics.md

```diff
@@ -0,0 +1,60 @@
+# RocksDB statistics
+
+Nebula Graph uses RocksDB as the underlying storage. This topic describes how to collect and show the RocksDB statistics of Nebula Graph.
+
+## Enable RocksDB
+
+By default, the function of RocksDB statistics is disabled. To enable RocksDB statistics, you need to:
+
+1. Modify the `--enable_rocksdb_statistics` parameter as `true` in the `storaged.conf` file. The default path of the configuration file is `/use/local/nebula/etc`.
+
+2. Restart the service to make the modification valid.
+
+## Get RocksDB statistics
+
+Users can use the built-in HTTP service in the storage service to get the following types of statistics. Results in the SON format are supported.
```

Comment:
JSON format

And please add this topic to the mkdocs.yml file. And the Chinese version.